go.mod

```
@@ -0,0 +1,11 @@
+module lolidelgado/github-users
```

Just a recommendation; normally the module is named after the github repo path.

This is done this way because maybe in a future someone decides to import your project, and as the name is `lolidelgado/github-users`, then it will lead to a failure to fetch this route.

In this case, the name should be:

```suggestion
module github.com/lolidelgado/ondemand-go-bootcamp
```

---

Module name updated, thanks for the recommendation!